### ISSUE TITLE
View public key from Trezor device

### ISSUE THREAD

**Issue by s-a-y** [stellar-expert/albedo#19]
In the playground I tried to show public key from account linked to trezor,
at first browser blocked "double" pop-up, but after I enabled it and confirmed hash on the device (which doesn't really tell user what's going on, so on-screen guidance would be very helpful)
Then it failed
<img width="441" alt="Screen Shot 2020-08-04 at 11 37 50 AM" src="https://user-images.githubusercontent.com/1667781/89243341-0ef23d80-d647-11ea-9743-e613fa2ecb08.png">
<img width="524" alt="Screen Shot 2020-08-04 at 11 38 05 AM" src="https://user-images.githubusercontent.com/1667781/89243344-10bc0100-d647-11ea-9889-4d819716c07a.png">

**Comment by orbitlens** [comment]
Yes, I'm working on this. There is a problem with signing/transactions processing on Trezor wallets. TrezorConnect doesn't support direct ED25519 signatures for raw binary data (which would be laso a convenient fallback for newly added `BumpFeeTransaction` type and muxed accounts), so I'm looking for alternative solutions. I asked a question [here](https://github.com/trezor/connect/issues/634), still waiting for an answer.

Probably, it's worth disabling Trezor login option for now.
